src/DataStructures/Tags/TempTensor.hpp

```diff
@@ -124,5 +124,33 @@ using TempabC = TempTensor<N, tnsr::abC<DataType, SpatialDim, Fr>>;
 template <size_t N, size_t SpatialDim, typename Fr = Frame::Inertial,
           typename DataType = DataVector>
 using Tempijaa = TempTensor<N, tnsr::ijaa<DataType, SpatialDim, Fr>>;
+
+// Temporary tensor types used in the file `ApparentHorizons/StrahlkorperGr.cpp`
+template <size_t N, size_t SpatialDim, typename Fr = Frame::Inertial,
+          typename DataType = DataVector>
+using Temp_hessian =
```

Comment:
I'm wondering if these should be named differently, since its not clear from the name that `Temp_hessian` has indices that are 2 dimensional. And also `Temp_hessian` seems to mix snake_case and CamelCase.

We already have a convention in tensor names that a,b,c,... = spacetime indices and i,j,k... = spatial indices.  Maybe we should adopt the convention that x,y,z are 2-dimensional indices.  In that case, `Temp_hessian` could be named `TempIxy`, `Temp_surface_dual_basis` could be `Tempxi`, and `Temp_grad_spatial_metric` could be `Tempiix`.

We probably want others to weigh in on the names before renaming these quantities.